modules/microshift-greenboot-testing-workload-script.adoc

```diff
@@ -0,0 +1,47 @@
+//Module included in the following assemblies:
+//
+//* microshift_running-applications/microshift-greenboot-workload-scripts.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-greenboot-test-workload-health-check-script_{context}"]
+= Testing a workload health check script
+
+.Prerequisites
+
+* You have root access.
+* You have installed a workload.
+* You have created a health check script for the workload.
+* The {product-title} service is enabled.
+
+.Procedure
+
+. To test that greenboot is running a health check script file, reboot the host by running the following commands:
++
+[source, terminal]
+----
+$ sudo reboot
+----
+//Q: Placeholder, please verify command in this case
+
+. Examine the output of greenboot health checks by running the following command:
++
+[source, terminal]
+----
+$ sudo journalctl -o cat -u greenboot-healthcheck.service
+----
+
+[NOTE]
+====
+{product-title} core service health checks run before the workload health checks.
+====
+
+//Q: What sample output can we provide? This is from the fedora docs
+
+. Example output
```

Comment:
Can you double check the output of the journalctl -o cat -u greenboot-healthcheck.service command from above ? My experience has been the Boot status message appears when I log into the system, not from the journalctl output.

Reply:
Nice catch!

Reply:
@ggiguash and @dhensel-rh Please clarify what changes, if any, are needed to this doc.